setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 
 install_requires = [
-    'bintrees==2.0.7',
+    'sortedcontainers>=1.5.9',
```

Comment:
In my personal projects, I don't mind setting `>=`... but given the broad use of this library, I'd recommend being more cautious and set either:

1) `sortedcontainers==1.5.9`
2) `sortedcontainers<2`

... to minimize chances of breaking changes.